src/platform/glfw/glfw_platform.cpp

```
@@ -18,6 +18,12 @@ namespace {
 
 namespace moth_graphics::platform::glfw {
 
+    Platform::Platform() = default;
```

Clang (14.0.6)

[error] constructor does not initialize these fields: m_context

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)